source/includes/gcp/_target_proxies.md

```diff
@@ -84,13 +89,62 @@ Retrieve a target proxy in an [environment](#administration-environments).
 | -------------------------------- | ------ |
 | `creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format. |
 | `id`<br/>*UUID*                  | The id of the backend service. |
-| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy, compute#targetHttpsProxy, compute#targetSslProxy or compute#TargetTcpProxy for target proxies.|
+| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy, compute#targetHttpsProxy, compute#targetSslProxy or compute#TargetTcpProxy for target proxies. |
 | `name`<br/>*string*              | Name of the resource. |
 | `selfLink`<br/>*string*          | Server-defined URL for the resource. |
 | `shortUrlMap`<br/>*string*       | A short version of the URL map. |
-| `type`<br/>*enum*                | A short version of the kind of resource. Either HTTP, HTTPS, SSL or TCP|
+| `shortCertificates`<br/>*string* | A short version of the SSL certificates. |
+| `type`<br/>*enum*                | A short version of the kind of resource. Either HTTP, HTTPS, SSL or TCP. |
 | `urlMap`<br/>*string*            | URL to the UrlMap resource that defines the mapping from URL to the BackendService. |
 
+<!-------------------- CREATE A TARGET PROXY -------------------->
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/targetProxies"
+
+# Examples:
+```
+
+```json
+{
+   "name": "targetProxyName",
+   "type": "HTTPS",
+   "shortUrlMap": "urlMapName",
+   "shortCertificates": "sslCertificateName"
+}
+```
+
+```json
+{
+   "name": "targetProxyName",
+   "type": "HTTPS",
+   "shortBackend": "backendServicesName",
+   "shortCertificates": "sslCertificateName"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/targetProxies</code>
+
+Create a new target proxy
+
+| Attributes | &nbsp;|
+| --- | --- |
+| `name`<br/>*string*              | Name of the resource. |
+| `shortBackend`<br/>*string*      | A short version of the backend service. |
+| `shortCertificates`<br/>*string* | A short version of the SSL certificates. |
+| `shortUrlMap`<br/>*string*       | A short version of the URL map. |
+| `type`<br/>*enum*                | A short version of the kind of resource. Either HTTP, HTTPS, SSL or TCP. |
+
+<aside class="notice">
```

Comment:
Some of these notices are not needed if the variables above are but in Required/Optional sections. See example picture above.

Reply:
Done.